### PR TITLE
Mail refactor: part 4

### PR DESCRIPTION
**Pull request by mzylowski**
Related with #490

Work in progress. When finished #603 and #612 will be fixed.

### CODE REVIEW

**Comment by mzylowski** [comment]
I decided to remove #612 from scope of this PR (wating for confirmation mail from COG).

@kojoty PTAL

File: newcache.php [hunk 1]
```diff
@@ -767,27 +769,9 @@
...
-
-                    $email_headers = "Content-Type: text/plain; charset=utf-8\r\n";
-                    $email_headers .= "From: $site_name <$octeam_email>\r\n";
-                    $email_headers .= "Reply-To: $octeam_email\r\n";
-                    $octeam_email = $octeam_email;
-
-                    //send email to octeam
-                    mb_send_mail($octeam_email, tr('rrActivateCache_07') . ": " . $name, $email_content, $email_headers);
+                if ($needs_approvement) { // notify OC-Team that new cache has to be verified
+                    EmailSender::sendNotifyAboutNewCacheToOcTeam(__DIR__ . '/tpl/stdstyle/email/oc_team_notify_new_cache.email.html',
```
Comment (kojoty) [inline newcache.php]: small note: in my opinion email templates should be set inside EmailSender module - I assume this method sends only one kind of messages

Comment (mzylowski) [inline newcache.php]: I will apply this after merging #918 into master for all e-mails.

**Comment by mzylowski** [comment]
I removed "To made EmailSender shorter.." commit from PR.

Merging.

**Pull request merged**